ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -304,6 +306,35 @@ def _llmobs_set_meta_tags_from_similarity_search(
                 except TypeError:
                     log.warning("Failed to serialize similarity output documents to JSON")
 
+    def _llmobs_set_meta_tags_from_tool(
+        self,
+        span: Span,
+        tool_input: Union[str, Dict[str, object], object],
+        tool_output: object,
+        error: bool,
+    ) -> None:
+        span.set_tag_str(SPAN_KIND, "tool")
+        if tool_input is not None:
```

Comment:
Can we also add the following to the llmobs span?
- Span name: should be the name of the tool
- Metadata: should include tool description and anything else in `config` (including metadata/tags) if applicable

Reply:
- Isn't span name the same as what we set it in the apm side ? In this case it is the tool name.
- I'll add metadata with the tool info (description, tags, metadata) and config

Reply:
Ahh yes, I forgot we set the resource name on the span for all langchain llmobs spans.

For metadata, can we try appending the key/value pairs in the langchain `config["metadata", "tags"]` argument into the llmobs metadata field?